examples/scale/backend/backend.go

```diff
@@ -77,8 +80,14 @@ func Run() {
 	fe := pb.NewFrontendClient(feConn)
 
 	go doFetch(cfg, be)
-	go doAssign(be)
-	go doDelete(fe)
+
+	if activeScenario.ShouldAssignTicket {
```

Comment:
Naming: Should this be "BackendAssignsTickets", and "BackendDeletesTickets"?

There are scenarios where the Frontend deletes tickets, so making the distinction would be useful.